src/common/ecmdDefines.H

```diff
@@ -35,14 +35,11 @@
 // include the ecmd build defines that apply to each type
```

Comment:
This may break the meson build .. let me cherry pick these changes and run once

Reply:
This should allow you to remove most of the defines from your meson.build file.  The key to making this work is to ensure `src/common` is the first path in the include listing (`-I src/common`).  This will ensure our ecmdDefines.H gets used as opposed to the empty default one in eCMD.